sh_script/rudra.sh

```diff
@@ -11,36 +11,31 @@ if [[ $? != 0 ]]; then
     exit
 fi
 
-origin=`cat rust-toolchain`
-
-flag=false
-if [[ ! $origin =~ "nightly-2021-08-20" ]];then
-    flag=true
-    echo "nightly-2021-08-20" > rust-toolchain
-    echo $origin
+if [[ ! $(cat rust-toolchain) =~ "nightly-2021-08-20" ]];then
+    echo echo -e "\033[31m Now rudra version supports 2021-08-20 \033[0m"
+    exit
```

Comment:
- Please document why this version is required.
- You could create a variable to avoid duplicating `nightly-2021-08-20`.
- Should the error be redirected to `stderr`?
- Can this be `exit 1` to signify clearly that an error occurred?

Reply:
Done